python/podio_class_generator.py

```diff
@@ -482,13 +482,13 @@ def _needs_include(self, classname) -> IncludeFrom:
 
     return IncludeFrom.NOWHERE
 
-  def _create_selection_xml(self):
+  def _create_ROOT_dictionary_selection(self):
     """Create the selection xml that is necessary for ROOT I/O"""
     data = {'components': [DataType(c) for c in self.datamodel.components],
             'datatypes': [DataType(d) for d in self.datamodel.datatypes],
             'old_schema_components': [DataType(d) for d in
                                       self.old_datamodels_datatypes | self.old_datamodels_components]}
-    self._write_file('selection.xml', self._eval_template('selection.xml.jinja2', data))
+    self._write_file('LinkDef.h', self._eval_template('DictLinkDef.h.jinja2', data))
```

Comment:
I just realized that there is a check in `_write_file` that puts the `LinkDef.h` file into the include directory with all the other (public) headers. Given that is only necessary for dictionary generation, I would consider it an implementation detail that should rather go into the sources output folder, right?

Reply:
Yes, it should at least not appear in the install. Will try to fix that

Reply:
Done with the last commits